### DISCUSSION TITLE
7月2号购买了专业版到现在邮箱还没收到激活码

### DISCUSSION THREAD

**Discussion by jame-guo** [getlantern/lantern-forum-cn#701] [Q&A]
7月2号购买了专业版到现在邮箱还没收到激活码。按指引链接无法打开无法提交问题资料。请尽快帮忙解决！
![QQ截图20240704011246](https://github.com/getlantern/lantern-forum-cn/assets/174894417/7be20faa-9fdf-49ee-be23-97dd00a9a7df)
![QQ浏览器截图20240704010840](https://github.com/getlantern/lantern-forum-cn/assets/174894417/8045571e-9f88-4ba4-8329-4c38e3876298)
![Uploading QQ截图20240707002732.png…]()
@Mike777Khoo

**Comment by jame-guo** [comment]
![Uploading QQ截图20240707002732.png…]()

**Comment by MaxMKMax** [comment]
@jame-guo 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by jame-guo** [reply]
> @jame-guo 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您! https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
> 
> *备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

你好，按你的链接指引输入邮箱后无法打开链接。已附上405问题截图 
![QQ截图20240709015919](https://github.com/getlantern/lantern-forum-cn/assets/174894417/43814570-2e10-4436-bdbc-3a1a1b6c7f97)

**Comment by jame-guo** [comment]
你好，按你的链接指引输入邮箱后无法打开链接。


Jame
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月9日(星期二) 凌晨0:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 7月2号购买了专业版到现在邮箱还没收到激活码 (Discussion #701)





 
@jame-guo 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱.
 后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
 https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
 
*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Comment by MaxMKMax** [comment]
@jame-guo 您好. 请问您的问题解决了吗? 那个链接关掉了. 
请您提供我账号邮箱, 我这里重新手动帮您提交问题.

**Comment by jame-guo** [comment]
解决了，谢谢



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年7月29日(周一) 晚上11:52
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 7月2号购买了专业版到现在邮箱还没收到激活码 (Discussion #701)




 
@jame-guo 您好. 请问您的问题解决了吗? 那个链接关掉了.
 请您提供我账号邮箱, 我这里重新手动帮您提交问题.

**Reply by MaxMKMax** [reply]
@jame-guo 那就好. 谢谢您的回复. 感谢您支持蓝灯. 我们非常开心和荣幸有像您这样的用户.
